index.css

```diff
@@ -403,10 +409,62 @@ body {
   .bold:hover {
     color: #ec5242;
   }
+
+  .about-images {
+    clear: both;
+    display: table;
+    padding: 4%;
+  }
+
+  .imageleft {
+    padding: 5px;
+    position: relative;
+  }
+
+  .photo {
+    display: block;
+    width: 100%;
+    height: auto;
+  }
+
+  .overlay {
+    position: absolute;
+    bottom: 0;
+    left: 100%;
+    right: 0;
+    background-color: #ec5242;
+    overflow: hidden;
+    width: 0;
+    height: 100%;
+    transition: 0.5s ease;
+  }
+
+  .imageleft:hover .overlay {
+    width: 100%;
+    left: 0;
+  }
+
+  .overlay-text {
+    color: white;
+    font-size: 28px;
+    font-family: 'Cocogoose', sans-serif;
```

Comment:
The pages use the same fonts as from the original design ✔

Reply:
Done